pyfftw/interfaces/_utils.py

```diff
@@ -63,8 +63,7 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
 
     if calling_func in ('irfft2', 'irfftn'):
         # overwrite_input is not an argument to irfft2 or irfftn
-        args = (a, s, axes, planner_effort, threads, 
-                auto_align_input, auto_contiguous)
+        args = (planner_effort, threads, auto_align_input, auto_contiguous)
```

Comment:
This prob means the key on line 79 should be updated as well - `args[3:]` -> `args`.